src/components/BulkEditList/BulkEditListResult/BulkEditInApp/BulkEditInApp.js

```diff
@@ -92,6 +95,10 @@ export const BulkEditInApp = ({ title, onContentUpdatesChanged }) => {
     }]);
   };
 
+  const getIsTemporaryLocation = ({ option }) => option === OPTIONS.TEMPORARY_LOCATION;
+
+
+
```

Comment:
👀

Reply:
Ye, my bad. Removed this empty string already.